gax/src/streamingCalls/streaming.ts

```diff
@@ -522,4 +362,232 @@ export class StreamProxy extends duplexify implements GRPCCallResult {
       this.stream.cancel();
     }
   }
+
+  /**
+   * Creates a new retry request stream -
+   *inner arrow function "newMakeRequest" handles retrying and resumption
+   * @param {streamingRetryRequestOptions} opts
+   *   {request} - the request to be made if the stream errors
+   *   {retry} - the retry options associated with the call
+   * @returns {CancellableStream} - the stream that handles retry logic
+   */
+  newStreamingRetryRequest(
```

Comment:
Should this be public or private?

Reply:
totally can be private, I'll work on that